stl/inc/random

```diff
@@ -959,26 +963,23 @@ struct _Swc_traits { // traits for subtract_with_carry generator
     static constexpr _Mod_t _Mod = _Mx;
     static constexpr _Ty _Max    = static_cast<_Ty>(_Mx - 1);
 
-    static int _Get_wc() noexcept { // compute number of 32-bit words per element
-        int _Kx;
-
+private:
+    static constexpr int _Get_wc() noexcept { // compute number of 32-bit words per element
         if constexpr (_Mx == 0) {
-            _Kx = (8 * sizeof(_Ty) + 31) / 32;
-        } else { // compute number of 32-bit words required
-            unsigned long long _Val = 1ULL << 32;
-            for (_Kx = 1; 0 < _Val && _Val < _Mx; ++_Kx) {
-                _Val = _Val << 32;
-            }
+            return (8 * sizeof(_Ty) + 31) / 32;
+        } else if constexpr (_Mx > (1ULL << 32)) {
+            return 2;
+        } else {
+            return 1;
         }
-
-        return _Kx;
     }
 
+public:
     template <class _Gen>
     static _Cy_t _Reset(_Gen& _Gx, _Ty* _Ax, bool _Readcy) { // set initial values of _Ax from generator _Gx

```

Comment:
Why mot constexpr here too?

Reply:
This isn't calculating a compile-time constant.